docs/threat-model.md

```diff
@@ -0,0 +1,553 @@
+
+**Cost**: VERY HIGH - Requires advanced cryptographic attacks or sophisticated key compromise
+
+**Impact**:
+- **Eligibility Bias**: Unfair advantage in EB creation opportunities
+- **Centralization**: Could concentrate EB production in attacker's control
+- **Cryptographic Failure**: Indicates fundamental security breach requiring protocol changes
+
+**Assets Affected**: Blockchain Safety, Decentralization Properties
+
+#### T7: Stake Grinding
+**Description**: Attacker manipulates stake distribution to influence voting power or EB eligibility through strategic delegation patterns.
+
+**Prerequisites**:
+- Significant economic resources to acquire or control stake
+- Knowledge of stake distribution update timing (epoch boundaries)
+- Ability to coordinate multiple delegation transactions
+
+**Attack Vector**:
+1. Accumulate stake through purchase or rental
+2. Time delegation changes to maximize impact on upcoming epochs
+3. Create multiple pools to spread influence while maintaining control
+4. Coordinate with other actors to amplify manipulation effects
+
+**Cost**: HIGH - Requires substantial capital investment in stake acquisition
+
+**Impact**:
+- **Voting Bias**: Disproportionate influence over EB certification
+- **Eligibility Manipulation**: Increased EB creation opportunities
+- **Centralization**: Concentration of power despite appearing decentralized
+
+**Assets Affected**: Decentralization Properties
+
+### 3.3 Transaction-Level Threats
+
+#### T8: Transaction Availability Attack
+**Description**: Attacker creates EBs referencing unavailable transactions to waste network resources and disrupt certification.
+
+**Prerequisites**:
+- EB creation eligibility (via VRF)
+- Control over transaction propagation to specific network segments
+- Coordination between transaction submission and EB creation
+
+**Attack Vector**:
+1. Submit transactions to limited network segments
+2. Create EB referencing these transactions before full propagation
+3. Voting nodes cannot verify transaction availability, preventing certification
+4. Forces futile transaction fetching attempts across network
+
+**Cost**: LOW - Minimal beyond normal EB creation costs
+
+**Impact**:
+- **Resource Waste**: Network bandwidth consumed fetching unavailable transactions
+- **Throughput**: Temporary reduction when EBs fail certification
+- **Operational**: SPO resources wasted on failed validation attempts
+
+**Assets Affected**: High Throughput, Operational Sustainability
+
+#### T9: Transaction Front-Running
+**Description**: EB producers observe profitable transactions and reorder or insert their own transactions to extract value before the original transaction executes.
+
+**Prerequisites**:
+- EB creation eligibility
+- MEV (Maximal Extractable Value) opportunities in transaction sets
+- Knowledge of transaction dependencies and profitable patterns
+
+**Attack Vector**:
+1. Monitor mempool for profitable transaction patterns
```

Comment:
Is this really different from the situation with Praos?

Reply:
No, but that does not make it less of a threat and I wanted to cover a range of threats which had been in discussion lately. What's your suggestion here?

Reply:
For this document we don't need to change anything, but when representation Leios we might want to clearly distinguish three categories of threats:

1. Exists in Praos and similarly exists in Leios.
2. Exists in Praos but more serious in Leios.
3. Does not exist in Praos but does exist in Leios.